lib/index.js

```diff
@@ -90,6 +89,7 @@ Appboy.prototype.identify = function(identify) {
   var phone = identify.phone();
   var traits = clone(identify.traits());
 
+  window.appboy.changeUser(userId);
```

Comment:
I feel like if there's no harm, we should probably guard this with `userId` being present incase someone uses `.identify()` without userId just to cache traits (albeit that is bad tracking implementation). 

Also confirming that calling this multiple times w same value in same page load is innocuous.